package.json

```diff
@@ -10,7 +10,7 @@
   "scripts": {
     "start": "nodemon server/index.js",
     "watch": "webpack serve --open",
-    "build": "webpack",
+    "build": "webpack --watch",
```

Comment:
Don't forget to switch this back for production.
In mine, I have a `server-start` because I kept forgetting I had the `--watch` flag on it and trying to build.

Reply:
OK, cool. I just added a script like yours so that I avoid that problem. Thanks!